src/lumigo_opentelemetry/instrumentations/pymongo/__init__.py

```diff
@@ -10,8 +17,43 @@ def check_if_applicable(self) -> None:
 
     def install_instrumentation(self) -> None:
         from opentelemetry.instrumentation.pymongo import PymongoInstrumentor
+        from pymongo.monitoring import (
+            CommandStartedEvent,
+            CommandSucceededEvent,
+            CommandFailedEvent,
+        )
+
+        def request_hook(span: Span, event: CommandStartedEvent) -> None:
+            with lumigo_safe_execute("pymongo_request_hook"):
+                span.set_attribute("db.statement", event.command_name)
+                if isinstance(event, CommandStartedEvent):
+                    span.set_attribute(
+                        "db.request.body",
```

Comment:
is this standard? i feel like db statements should be entirely in the statement itself, as it is the command seems to usually be the name of the span

Reply:
This is the convention the we invent in order to standardized the API of DB instrumentations. Just like `db.response.body` in redis :)